### PR TITLE
Update from OpensslIBM support repo.

### PR DESCRIPTION
**Pull request by smcmahonibm**
Includes DH Q checking and PKCS12 fixes.

### CODE REVIEW

**Review by lsutherland-ibm — APPROVED** [review]

**Comment by smcmahonibm** [comment]
The CI fails because of the other error so can ignore that for this PR.

File: icc/icc.rc [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+#include "iccversion.h"
+/////////////////////////////////////////////////////////////////////////////
+#undef APSTUDIO_READONLY_SYMBOLS
+
+/////////////////////////////////////////////////////////////////////////////
+// English (Australia) resources
+
+#if !defined(AFX_RESOURCE_DLL) || defined(AFX_TARG_ENA)
+#ifdef _WIN32
+LANGUAGE LANG_ENGLISH, SUBLANG_ENGLISH_AUS
```
Comment (lsutherland-ibm) [inline icc/icc.rc:20]: Note that there are a couple references to locale

File: icc/icc.rc [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+            VALUE "Comments", "\0"
+            VALUE "CompanyName", "IBM\0"
+            VALUE "FileDescription", ICC_DESCRIPTION1
+            VALUE "FileVersion", ICC_PRODUCT_VERSION
+            VALUE "InternalName", "ICC\0"
+            VALUE "LegalCopyright", "Licensed Materials - Property of IBM ICC (C) Copyright IBM Corp. 2002, 2018 All Rights Reserved.  US Government Users Restricted Rights - Use, duplication or disclosure restricted by GSA ADP Schedule Contract with IBM Corp.\0"
+            VALUE "LegalTrademarks", "IBM\0"
+            VALUE "OriginalFilename", "icc.lib\0"
+            VALUE "ProductName", "IBM ICC"
+            VALUE "ProductVersion", ICC_PRODUCT_VERSION
```
Comment (lsutherland-ibm) [inline icc/icc.rc:56]: This LegalCopyright might not be the one we want for this repo

Comment (smcmahonibm) [reply icc/icc.rc]: fixed

**Review by lsutherland-ibm — APPROVED** [review]

**Review by robindubeyibm — APPROVED** [review]
Looks Good to Me

**Pull request merged**